src/index.js

```diff
@@ -89,8 +89,10 @@ const isTest = !!process.env.TEST_DATABASE;
 const isProduction = !!process.env.DATABASE_URL;
 const port = process.env.PORT || 8000;
 
-sequelize.sync({ force: isTest || isProduction }).then(async () => {
+connectDb().then(async () => {
   if (isTest || isProduction) {
+    // resetDb
+    await Promise.all([models.User.remove({}), models.Message.remove({})]);
```

Comment:
Out of curiosity: Is it necessary to pass an empty object when deleting the entire table?

Reply:
yes it's necessary. the empty object {} is the filter argument when empty all documents would be removed. for reference https://docs.mongodb.com/v3.2/tutorial/remove-documents/#delete-all-documents